### ISSUE TITLE
Individual page creation: "Save and next" styling

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-in-action/platform#937]
To minimize user movements, prioritize "Save and next" and use the primary button styling for it and make the "Save" secondary.

**Comment by cherylhjli** [comment]
@sepidehshahi can you tell me more about what minimizing user movements mean?

**Comment by sepidehshahi** [comment]
Currently "Save" button is the primary button, so user first clicks the save button. Then, they realize that they should have clicked the save and next to go to the next step. This will create extra movements for head tracker and switch users. If the "Save and next" was primary they would have selected it first and went to the next step initially. 

Once the page is published, we can make the "Save" button the primary button, but during page creation, it seemed that "Save and next" is the primary action.

**Comment by cherylhjli** [comment]
Thanks Sepi! I updated this on all the members pages and the project page. 

So on any page with a "save" and "save and next" (and even a "save and previous"), "save and next" is the primary: https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1125%3A5877

On the last step, "save" is the primary: https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1229%3A13916

Editing a published page, "save" is primary: https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=2472%3A63298

**Comment by greatislander** [comment]
I've implemented this except for this last point:

> Editing a published page, "save" is primary

I don't think there's a good case for changing which button is primary if someone is editing a published page. Besides, if they unpublish it, would we then switch back to having "Save and next" be primary? Would it be clear to someone using the website why the buttons are changing? I think "Save and next" should always be primary for consistency.
